data/common.yaml

```diff
@@ -1 +1,2 @@
---- {}
+---
+k8s::sysconfig_path: '/etc/sysconfig'
```

Comment:
instead of adding it here, could you move the data to the init.pp? that enables puppet-strings to pick it up.

Reply:
If I remember correctly, it was in hiera to support doing lookup without including the main `k8s` class, though it seems the code that required that isn't present anymore.